src/tbots.py

```diff
@@ -76,8 +76,10 @@
     if args.debug_build:
         command += ["-c", "dbg"]
 
-    # If its a binary, then run under gdb
-    if args.action in "run" and args.debug_build:
+    # If its a binary, then run under gdb. We need to special case thunderscope
+    # because it relies on --debug_simulator and --debug_simulator and prompts
+    # the user to run the command under gdb instead.
+    if args.action in "run" and args.debug_build and "thunderscope" not in target:
```

Comment:
shouldn't debug_build cause the script to pass in debug_fullsystem to thunderscope if `-t` (thunderscope) is passed in?

Reply:
yeah i see, changed it to work

 ./tbots.py -d run goalie_test --debug_fullsystem